### ISSUE TITLE
NVDA does not read outgoing messages in messenger.com on microsoft edge

### ISSUE THREAD

**Issue by Fatihyuksek** [nvaccess/nvda#8347]
### Steps to reproduce:
> A list of the steps you take to demonstrate the problem. 
>
> Example:
>
> 1. Open Chrome
> 2. Browse to www.google.com
> 3. Type "Hello"
> 4. Notice an error sound when enter is pressed.

> Please also remember to attach a zip of any files required to reproduce the issue.

### Expected behavior:
> Tell us what should happen.

### Actual behavior:
> Tell us what happens instead.

### System configuration:
NVDA version:
> Example: next-14027,c80e529f

NVDA Installed or portable:

Other information:
> Example: Running in a VM

Windows version:
> Example: Windows 10 Version 1607 Build 14393.1066

Name and version of other software in use when reproducing the issue:

### Other questions:

Does the issue still occur after restarting your PC?

Have you tried any other versions of NVDA?
> Please list them and the result
I heard that in other browsers NVDA would read incoming and outgoing messages in messenger.com but that doesn't work for Microsoft edge. I haven't tested and tried another browser so I don't know whether NVDA reads the messages on other browsers while chatting in messenger.com.

**Comment by ehollig** [comment]
@Fatihyuksek, could you test and see if these notifications are being read in Firefox or Chrome on messenger.com?

**Comment by Adriani90** [comment]
Ok I don't know if this is a web authoring bug, I think outgoing messages are not exposed as live region when you press enter to send it. Outgoing messages are not reported in any browser.

I noticed in Firefox however that only the first two words or so of the incoming messages are reported. This is quite strange. It seems in Chrome and Edge Chromium incoming messages are reported completely. 

messenger.com is the web app for the facebook messenger.

@josephsl, @MarcoZehe I think you also have Facebook account. Right? Could you test this as well? Thanks.
